### ISSUE TITLE
iOS music from other app stopped when playSoundFile is triggered

### ISSUE THREAD

**Issue by ansoncsh** [johnsonsu/react-native-sound-player#72]
Hi, I am using this library for playing sound track **SoundPlayer.playSoundFile()** in my app.
Once a sound track is played, the music running by other apps would be stopped.
Is there any configuration to prevent it?

Thank you for help!

**Comment by johnsonsu** [comment]
Yes, it should be possible with so native code change. More specifically, we need to change [Audio Session Category](https://developer.apple.com/documentation/avfoundation/avaudiosession#//apple_ref/doc/constant_group/Audio_Session_Categories). I can look into this when I'm free but you can also try tackling it! A PR is greatly appreciated!

**Comment by BrianH2** [comment]
@johnsonsu thank you for this package 🙌 I'm having the same issue, so took a stab at it in #127. I'm a native noob so all feedback is welcome. Can you take a look when you have a chance please?
